app/src/components/edit-tabs-module/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { IFormErrors, IFormValues } from "../../pages/edit";
+import Contacts from "./contacts";
+import Main from "./main";
+import { Dayjs } from "dayjs";
```

Comment:
Please, follow the order of imports.  Lines with libraries come first, then local files